recipes/conspire/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "conspire" %}
+{% set python_min = "3.9" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/conspire-{{ version }}.tar.gz
+  sha256: a071b8a9be01a782e1a8abef2414f653fde4067bc3806d71f774a16765c6094e
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('rust') }}
+  host:
+    - maturin
+    - pip
+    - python {{ python_min }}
+  run:
+    - libgcc
+    - numpy
+    - python >={{ python_min }}
+
+test:
+  commands:
+    - pip check
+    - pytest
+  imports:
+    - conspire
+  requires:
+    - pip
+    - pytest
+    - python {{ python_min }}
+  source_files:
+    - pyproject.toml
+    - tests/
+
+about:
+  home: https://mrbuche/github.io/conspire
+  dev_url: https://github.com/mrbuche/conspire
```

Comment:
It's probably more accurate to change this to `dev_url: https://github.com/mrbuche/conspire.py`